lib/iris/coords.py

```diff
@@ -671,6 +679,32 @@ def _bounds_setter(self, bounds):
 
     bounds = property(_bounds_getter, _bounds_setter)
 
+    def _climatology_getter(self):
+        """
+        A boolean that controls whether the coordinate bounds represent a
+        climatological, rather than a normal period.
+
+        Always reads as False if there are no bounds.
+        On set, the input value is cast to a boolean, and an exception is
+        raised if there are no bounds.
+
+        """
+        result = False
+        if self.has_bounds():
+            result = self._bounds_are_climatological
+        return result
```

Comment:
@trexfeathers This could all neatly boil down to:
```python
return self._climatology if self.has_bounds() else False
```

Reply:
👍